### PR TITLE
docs: removed `$` from code snippets

### PR DESCRIPTION
**Pull request by iamayushdas**
removed `$` from code snippets so that while copying from copy button on copy buttons doesn't copy trailing dollars

@spacewander @Yiyiyimu

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
Hi @iamayushdas could you also fix the ones in README

**Comment by iamayushdas** [comment]
Sure

**Comment by iamayushdas** [comment]
cc @Yiyiyimu done

File: README.md [hunk 1]
```diff
@@ -11,36 +11,37 @@
 # Assign Apache release version number to variable `APISIX_VERSION`, for example: 2.6. The latest version can be find at `https://github.com/apache/apisix/releases`
 
 export APISIX_VERSION=2.6
+
 # alpine
-$ make build-on-alpine
+make build-on-alpine
```
Comment (Yiyiyimu) [inline README.md:16]: I believe we should separate command for alpine and centos into to code block, since people won't use them at the same time

Comment (Yiyiyimu) [reply README.md:16]: cc @iamayushdas do you have time working on this

Comment (iamayushdas) [reply README.md:16]: Yes i have

**Comment by leslie-tsang** [comment]
Any update for this PR? @iamayushdas

**Pull request closed without merging**